wikitemplate-minorCRbumpAndroid.md

```diff
@@ -0,0 +1,75 @@
+## Installer
+
+- [ ] Check that installer is close to the size of the last release
+- [ ] Check the Brave version in About and make sure it is EXACTLY as expected
+
+## Custom tabs
+
+- [ ] Make sure Brave handles links from Gmail, Slack
+- [ ] Make sure Brave works as custom tabs provide with Chrome browser
+- [ ] Ensure custom tabs work even with sync enabled/disabled
+
+## Developer Tools
+
+- [ ] Verify you can inspect sub-links via dev tools
```

Comment:
We dont need this one either for minor bump

Reply:
@bbondy mentioned that we might need this one as it might be affected by the actual CR version number. @bbondy thoughts?

Reply:
maybe not I just thought there was something about devtools that can change with build numbers. @SergeyZhukovsky can say if it's ok to remove.

Reply:
The problems used to be on tabs repo only and only on major bumps, we have never seen any issues on brave-core. It's totally fine to remove it.

Reply:
Sounds good 👍 Thanks @bbondy @SergeyZhukovsky. @srirambv will remove as per your suggestion.

Reply:
Removed 👍